src/main/java/mate/project/service/BookService.java

```diff
@@ -4,11 +4,12 @@
 import mate.project.dto.BookDto;
 import mate.project.dto.BookSearchParameters;
 import mate.project.dto.CreateBookRequestDto;
+import org.springframework.data.domain.Pageable;
```

Comment:
There is an import of the 'Pageable' class from the 'org.springframework.data.domain' package, but the code doesn't seem to handle cases where the 'Pageable' object is null.